python-sdk/tests/prompts/test_fetch_policies.py

```
@@ -0,0 +1,534 @@
+                )
+
+            # Verify no HTTP calls were made
+            mock_request.assert_not_called()
+
+    finally:
+        os.chdir(original_cwd)
+
+
+@pytest.mark.integration
+def test_cache_ttl_returns_cached_version_before_expiration(
+    cli_prompt_setup: Path, clean_langwatch
```

Ruff (0.14.8)

Unused function argument: `clean_langwatch`

(ARG001)